### DISCUSSION TITLE
Approved and rejected at the same time

### DISCUSSION THREAD

**Discussion by TaughtMe** [community/community#112362] [GitHub Education]
### Hi everyone,

I got an E-Mail that I have been rejected für GitHub Edu but in the submission field on GitHub Benfits it says that I have been aprooved. Does one normally get an E-Mail when they have been approved?

Thanks in advnace.

**Comment by samus-aran** [comment]
Hi @TaughtMe 👋 

That doesn't seem right! Perhaps there is a time delay on email notifications. It's best to refer to the "latest" or top most application status on your GitHub Education application. Please check the status of your application [here](https://education.github.com/discount_requests/pack_application).

**Reply by TaughtMe** [reply]
Thanks for your reply it just took a couple of days and the mail might have been from an older request.
